docs/v0.4/invokers/command.md

```diff
@@ -4,12 +4,42 @@ title: Command Function Invoker
 sidebar_label: Command
 ---
 
-## Command Function Invoker
+Command functions will be invoked using a [Command Function Invoker](https://github.com/projectriff/command-function-invoker) that is provided by riff when you build the function.
 
-<!-- TODO replace the redirect with real content -->
+The *command function invoker* provides a host for functions implemented
+as a single executable command (be it a shell script or a binary).
+It accepts HTTP requests and invokes the command for each request.
 
-[Learn about riff's Command Function Invoker](https://github.com/projectriff/command-function-invoker)
+Communication with the function is done via `stdin` and `stdout`.
+Functions can log by writing to `stderr`.
 
-<script type="text/javascript">
-  window.location.href = 'https://github.com/projectriff/command-function-invoker';
-</script>
+For each invocation, functions are expected to read stdin until the end of the stream (EOF) and provide a result on stdout.
+
+Correct function execution is assumed if the exit code is zero. Any other value indicates an error.
+
+## Creating a simple command function
+
+This example uses the sample [command-wordcount](https://github.com/projectriff-samples/command-wordcount) function from projectriff-samples on GitHub. It consists of a single executable file named wordcount.sh with the following content:
+
+```bash
+#!/bin/bash
+
+tr [:punct:] ' ' | tr -s ' ' '\n' | tr [:upper:] [:lower:] | sort | uniq -c | sort -n
+```
+
+### Building the command function
+
+> NOTE: If you are creating command functions on Windows then you can't set the excute flag on a local file. This means that local-path builds will not work, you need to build the function from a Git repository. Before commiting your function file to a Git repository you should set the excute flag using the following Git command: `git update-index --chmod=+x wordcount.sh`.
```

Comment:
Local path builds are disabled for Windows universally. It's not specific to the command invoker anymore.